app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -75,6 +78,7 @@ def had_pending_eps?
   def pending_eps?
     pending = open_claims.any? { |claim| claim['base_end_product_code'] == '020' }
     save_metadata(offramp_reason: 'pending_ep') if pending
+    save_metadata(pending_eps: pending)
```

Comment:
So that we know the value should be a boolean:
```suggestion
    save_metadata(has_pending_eps: pending)
```